### ISSUE TITLE
Docker setup doesn't work; `make install` should be after `make jekyll`?

### ISSUE THREAD

**Issue by estark37** [google/badssl.com#166]
Docker setup doesn't work out of the box:
```bash
$ make docker
sudo docker build -t badssl .
...
Sending build context to Docker daemon 7.457 MB
Step 1 : FROM ubuntu:16.04
 ---> 17b6a9e179d7
Step 2 : MAINTAINER April King <april@twoevils.org>
 ---> Using cache
 ---> 496c4e159766
Step 3 : EXPOSE 80 443
 ---> Using cache
 ---> 43f7090ccd2a
Step 4 : RUN apt-get update && apt-get install -y     build-essential     git     make     nginx     ruby     ruby-dev
 ---> Using cache
 ---> 9bbe69580e2d
Step 5 : RUN gem install jekyll
 ---> Using cache
 ---> b24fb3692d8b
Step 6 : ADD . badssl.com
 ---> 79911ecdaab9
Removing intermediate container c7935c0d8c5f
Step 7 : WORKDIR badssl.com
 ---> Running in 5cb3d08bd206
 ---> 7dd27573c74c
Removing intermediate container 5cb3d08bd206
Step 8 : RUN make install
 ---> Running in 1fd64c9b4899
./_site/certs/cert-generator/cert-generator.sh y
make: ./_site/certs/cert-generator/cert-generator.sh: Command not found
make: *** [keys] Error 127
```

It works if I change the Dockerfile to run `make jekyll` after `make install`, but not sure if that's the right fix or not. @Hoverbear

**Comment by Hoverbear** [comment]
@estark37 Hi, you're quite right about this. The change was made in #156 but never on `master`.

**Comment by lgarron** [comment]
Yeah, stuff is currently a little too broken (#162), and the time I've spent on it hasn't been enough to fix it.

@Hoverbear, could you isolate the fix from #156 separately? I haven't been able to justify to myself spending time on #156 while I can't even set up a VM from scratch. :-(

**Comment by Hoverbear** [comment]
@lgarron I did. :) Theyre in #167. 

I believe @marumari was reviewing #156 but she's otherwise occupied with a much more important little human. :)
